manager/watcher/raw/raw.go

```diff
@@ -135,7 +135,8 @@ func (self *rawContainerWatcher) watchDirectory(dir string, containerName string
 	for _, entry := range entries {
 		if entry.IsDir() {
 			entryPath := path.Join(dir, entry.Name())
-			_, err = self.watchDirectory(entryPath, path.Join(containerName, entry.Name()))
+			subcontainerName := path.Join(containerName, entry.Name())
+			alreadyWatchingSubDir, err := self.watchDirectory(events, entryPath, subcontainerName)
```

Comment:
Does this correctly handle the case when there are more than 2 layers created simultaneously? I'm not sure what entryPath & containerName would be in that case.

Reply:
Yes, the recursive call to watchDirectory will run through that directory's subdirectories and end up adding the container.